### PR TITLE
Added JenkinsRule.showAgentLogs utility

### PR DESCRIPTION
**Pull request by jglick**
Developed during JEP-210 work but could be broadly useful.

### CODE REVIEW

File: src/main/java/org/jvnet/hudson/test/JenkinsRule.java [hunk 1]
```diff
@@ -988,6 +990,53 @@ public void waitOnline(Slave s) throws Exception {
         }
     }
 
+    /**
+     * Same as {@link #showSlaveLogs(Slave, Map)} but taking a preconfigured list of loggers as a convenience.
+     */
+    public void showSlaveLogs(Slave s, LoggerRule loggerRule) throws Exception {
```
Comment (batmat) [inline src/main/java/org/jvnet/hudson/test/JenkinsRule.java]: I'm sorry to arrive late, but can you please rename to `showAgentLogs` instead? let's not keep adding new code about this.

Comment (jglick) [reply src/main/java/org/jvnet/hudson/test/JenkinsRule.java]: No strong feelings either way, just I was matching

* the type `Slave` used in the method signature
* the existing methods right next to this (`createSlave`, `createOnlineSlave`)

Comment (batmat) [reply src/main/java/org/jvnet/hudson/test/JenkinsRule.java]: Yeah, I know we'll be more inconsistent, but I think we need to start moving in that direction.

Comment (jglick) [reply src/main/java/org/jvnet/hudson/test/JenkinsRule.java]: OK, can change it.

Comment (batmat) [reply src/main/java/org/jvnet/hudson/test/JenkinsRule.java]: So, I won't block this alone if someone else is fine with adding more of these. But again I'd rather see us start removing this phrasing, even if this unfortunately raises a bit the code naming inconsistency.

**Pull request merged**